.github/renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "config:js-app",
+    ":gitSignOff",
+    "npm:unpublishSafe",
```

Comment:
I usually find 1 day to be enough via:

```json
"npm": {
    "stabilityDays": 1
},
```

It depends on your desired grouping and dependency count... If you don't have enough distinct groups all minor version bumps are bundled into a single PR. And this single PR can get constantly updated never reaching the "3 days old" mark, since there's always something new somewhere...

So either provide more distinct groups (for test libraries - `group:test`, for react+react types - `group:react`, etc) to lower the count of updated packages per PR or lower the days delay.

There's nothing wrong with trying it the way it is currently written in the PR, this is just my experience.

Reply:
@tumido fyi, `stabilityDays` has been renamed to `minimumReleaseAge`. I added `group:test` since `group:react` is added in `config:js-app`.